compose/nosqlbench-entrypoint.sh

```diff
@@ -17,62 +21,57 @@ test_conn zdm_tests_origin
 test_conn zdm_tests_target
 test_conn zdm_tests_proxy
 
-set -e
+set -xe
 
 echo "Creating schema"
 cat /source/nb-tests/schema.cql | cqlsh zdm_tests_proxy
 
 echo "Running NoSQLBench RAMPUP job"
-java -jar /nb.jar \
+java -jar /nb5.jar \
   --show-stacktraces \
-  /source/nb-tests/cql-nb-activity.yaml \
+  /source/nb-tests/cql_nb_activity.yaml \
   rampup \
-  driver=cqld3 \
   hosts=zdm_tests_proxy \
   localdc=datacenter1 \
-  errors=retry \
+  errors=counter,retry \
   -v
 
 echo "Running NoSQLBench WRITE job"
-java -jar /nb.jar \
+java -jar /nb5.jar \
   --show-stacktraces \
-  /source/nb-tests/cql-nb-activity.yaml \
+  /source/nb-tests/cql_nb_activity.yaml \
   write \
-  driver=cqld3 \
   hosts=zdm_tests_proxy \
   localdc=datacenter1 \
-  errors=retry \
+  errors=counter,retry \
   -v
 
 echo "Running NoSQLBench READ job"
-java -jar /nb.jar \
+java -jar /nb5.jar \
   --show-stacktraces \
-  /source/nb-tests/cql-nb-activity.yaml \
+  /source/nb-tests/cql_nb_activity.yaml \
   read \
-  driver=cqld3 \
   hosts=zdm_tests_proxy \
   localdc=datacenter1 \
-  errors=retry \
+  errors=counter,retry \
   -v
 
-echo "Running NoSQLBench VERIFY job on ORIGIN"
-java -jar /nb.jar \
-  --show-stacktraces \
-  --report-csv-to /source/verify-origin \
-  /source/nb-tests/cql-nb-activity.yaml \
-  verify \
-  driver=cqld3 \
-  hosts=zdm_tests_origin \
-  localdc=datacenter1 \
-  -v
+#echo "Running NoSQLBench VERIFY job on ORIGIN"
```

Comment:
Without the VERIFY step, this test is pretty much useless because this is what actually compares the stored data against the pseudo-random generated data.
I don't have the context why the NB is happening in this PR (if it's for some new feature or compatibility with newer C* versions), but if we could still run the old NB with the verify step in addition to this, until it's support in the latest NB version, I think it would be ideal.

Reply:
Understood. NB 5.21 doesnt support `verify` feature yet.